src/javascript/app/base/binary_loader.js

```diff
@@ -141,12 +142,22 @@ const BinaryLoader = (() => {
             loadActiveScript(config);
         }
         if (config.no_mf && Client.isLoggedIn() && Client.isAccountOfType('financial')) {
-            BinarySocket.wait('authorize').then(() => displayMessage(error_messages.no_mf()));
+            BinarySocket.wait('authorize').then(() => {
+                if (this_page === 'platforms') {
```

Comment:
can we have a separate flag for residence blocked rather than checking the name of the page?